src/components/Savings/SavingsModal/index.tsx

```diff
@@ -309,7 +295,7 @@ const SavingsModal = memo(
                                         <>
                                             <span>How much would you like to {type}</span>
                                             <SwapInput
```

Comment:
We are not using the new input/output here yet, to keep in-line with existing UI and legacy components
will be updated to native-base input/outputs when we get designs for these pages

Reply:
@L03TJ3 could we update to nativebase component keeping current design ?

Reply:
we potentially could, but that means creating two variants of the input/outputs, one with legacy styling, other with new styling.

Don't see the point in that, it will all be updated in due time and doesn't seem like our first and top priority now right?